jdbc-handler/src/main/java/org/apache/hive/storage/jdbc/conf/JdbcStorageConfigManager.java

```diff
@@ -45,6 +45,8 @@ public class JdbcStorageConfigManager {
   public static final String CONFIG_PWD_KEYSTORE = Constants.JDBC_KEYSTORE;
   public static final String CONFIG_PWD_KEY = Constants.JDBC_KEY;
   public static final String CONFIG_PWD_URI = Constants.JDBC_PASSWORD_URI;
+  public static final String CONFIG_JDBC_USERNAME = Constants.JDBC_USERNAME;
```

Comment:
@InvisibleProgrammer, do we need to duplicate the constants?

Reply:
No, we don't need to duplicate. I just now see that it already exists with another name, CONFIG_USERNAME. Let me remove the duplication.